docs/development/getting-started.rst

```diff
@@ -13,6 +13,18 @@ dependencies, install ``cryptography`` in ``editable`` mode. For example:
     $ pip install --requirement dev-requirements.txt
     $ pip install --editable .
 
+On OS X:
+
+You must have installed `OpenSSL`_ via `Homebrew`_ or `MacPorts`_ and must set ``CFLAGS`` and ``LDFLAGS`` environment variables before installing the ``dev-requirements.txt`` otherwise pip will fail with include errors. For example with `Homebrew`_:
+
+.. code-block:: console
+
+    $ env LDFLAGS="-L$(brew --prefix openssl)/lib" CFLAGS="-I$(brew --prefix openssl)/include" pip install --requirement ./dev-requirements.txt
+
+Alternatively for a static build you can specify ``CRYPTOGRAPHY_OSX_NO_LINK_FLAGS=1`` and ensure ``LDFLAGS`` points to the absolulte path for the `OpenSSL`_ libraries before calling pip.
```

Comment:
Still a typo here (absolute).

These lines also need to be < 80 chars each (outside of the code-block) or the `doc8` tests will fail. You can run the doc tests locally with `LDFLAGS="-L$(brew --prefix openssl)/lib" CFLAGS="-I$(brew --prefix openssl)/include" tox -e docs` if you want faster cycle times than our CI 😄